pipenv/core.py

```diff
@@ -2866,9 +2866,7 @@ def do_check(
             pypi_mirror=pypi_mirror,
         )
     if not quiet and not project.s.is_quiet():
-        click.echo(
-            click.style(decode_for_output("Checking PEP 508 requirements..."), bold=True)
-        )
+        click.echo(click.style("Checking PEP 508 requirements...", bold=True))
```

Comment:
And finally, please don't use `click.echo(click.style(...))`, instead use `click.secho`.

Reply:
Thanks! I just updated those references, and I kept others that are composed of styled and unstyled strings.